src/aura/CRLP_FilterGroup/CRLP_FilterGroup.cmp

```diff
@@ -270,7 +270,7 @@
                                                      required="true" label="{!v.labels.constant}" type="datetime-local"/>
                                 </aura:if>
                                 <aura:if isTrue="{!equals(v.filterRuleFieldType, 'number')}">
-                                    <lightning:input class="slds-p-around_small" value="{!v.activeFilterRule.value}"
+                                    <lightning:input aura:id="filterRuleField" class="slds-p-around_small" value="{!v.activeFilterRule.value}"
```

Comment:
Looks like this breaks the recent change to create a default filter rule for `Amount=null` to mimic legacy behavior.
![image](https://user-images.githubusercontent.com/801017/40665949-980d6526-632c-11e8-81f9-b4fed5ee89a5.png)